english/dig/cup-cocoa/cup-cocoa.md

```diff
@@ -2,8 +2,17 @@ You enjoy the warm cup of cocoa and you have a conversation with the big bird th
 Big Bird from Sesame Street. You ask him what he is doing in the forest and ask him if he would like
 a cup of "cocoa". He starts to correct you and says it is pronounced "co-co-aaaaaaaaa" and not "co-co".
 This pointless debate ensues for a few minutes until he concedes that it is indeed cocoa after you google
-the correct pronunciation. Then you ask him again what he is doing in the forest. 
+the correct pronunciation. Then you ask him again what he is doing in the forest.
 
 He says:
 
-"I am searching for delicious mushrooms. Do you have any?"
\ No newline at end of file
+"I am searching for delicious mushrooms. Do you have any?"
+
+You realize that this day has gotten out of hand.  Big bird is scary, the cocoa not longer tastes good, you want to go home.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.